presentation/backend/src/presentation-backend/RulesetEmbedder.ts

```diff
@@ -246,36 +251,39 @@ export class RulesetEmbedder {
     return rulesetList;
   }
 
-  private async getOrCreateRulesetModel(callbacks?: InsertCallbacks): Promise<DefinitionModel> {
-    const rulesetModel = this.queryRulesetModel();
+  private async getOrCreateRulesetModel(callbacks?: InsertCallbacks, subjectId?: Id64String): Promise<DefinitionModel> {
+    const subject = subjectId ? this._imodel.elements.getElement<Subject>(subjectId) : await this.getOrCreateSubject(callbacks);
+
+    const rulesetModel = this.queryRulesetModel(subject);
```

Comment:
The issue description isn't quite clear on whose parent the given `Subject` should be. It should be a parent for the `Subject` that we insert in `insertSubject`. When not specified, the parent should stay as is - the root `Subject`.

Also, I think we should handle the case when an ID of non-existing element is given - we should throw if we can't find a `Subject` wit that ID.

Reply:
Made described changes. 
> we should throw if we can't find a Subject wit that ID

I expect an error to be thrown at `imodel.getElement(parentSubjectId)`